PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.hdinsight/src/com/microsoft/azuretools/hdinsight/projects/MavenPluginUtil.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) Microsoft Corporation
+ * 
+ * All rights reserved. 
+ * 
+ * MIT License
+ * 
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files 
+ * (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, 
+ * publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, 
+ * subject to the following conditions:
+ * 
+ * The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+ * 
+ * THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF 
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR 
+ * ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH 
+ * THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+package com.microsoft.azuretools.hdinsight.projects;
+
+import com.microsoft.azuretools.core.utils.PluginUtil;
+
+public class MavenPluginUtil {
+	private static final String m2ePluginMarketplaceURL = "http://marketplace.eclipse.org/marketplace-client-intro?mpc_install=1774116";
```

Comment:
>http://marketplace.eclipse.org/marketplace-client-intro?mpc_install=1774116 [](start = 56, length = 75)

maybe we can try to use link forwarding here?

Reply:
Good point


---
In reply to: [136772785](https://github.com/Microsoft/azure-tools-for-java/pull/984#discussion_r136772785) [](ancestors = 136772785)

Reply:
Fixed

---
In reply to: [138527764](https://github.com/Microsoft/azure-tools-for-java/pull/984#discussion_r138527764) [](ancestors = 138527764,136772785)